### PR TITLE
adding mxnet embedding serving notebook

### PR DESCRIPTION
**Pull request by la-cruche**
*Issue #, if available:*

*Description of changes:*
Adding a new mxnet gluon notebook in the sagemaker-python-sdk section, showcasing use of custom inference functions to serve embeddings instead of a full classifier

### CODE REVIEW

File: sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
...1RKBHFPmPj5Ru7GLMSte0Qy7JYOKgycLoUSznq9Sa+7+O6IUkSAzGZ3BSqnKNYypPPJ3QHNZSwhCGH9FptZqYnQNIoDmWo7TYoDWmsXE1AifnV3/xpehs6p991OHnmPO+8eZlTl89w7UKHcqZEqxHwwx87jB43cZ2QBz5wnAPTM+SGbWxGuOfgAUbVOcLGCEppmwiDUjHF0buHMWSdTj0gCBW6/R4HDs+TTpvIkRhPFgRBEARBEARBEIR/CERpKNxyitEmCp0bhVwcYKUlfvhjD3Jt+VWWVmrk8mn6gxZREpFNZ0iZE4SRSzaXZnt9QEmf5dGHnqJSyZCyDXIFk3Qm4ujRo9R2WuzuLhNEHRRTYWe3yuKFNSRJwrAVwsinkC5x/fJ1Fi/77Gz0+OVf+m2yuSGyRQtN05AViElIkoRafRPPC+hqAeNjFcIk5nOf/z263YjdHYedrZBWKyYMFJIkwQ8SJEUllVFJCJFUH9fVgQTPken3HYLAQ1FDTDvB9300Rae2m9Bp+bz1+iLXFzfIl3RGxkv0ez65lImiSOTzeZzBAM/zCAOJYrFMFCU8+OCPE4Q9+ttH0EwD09JYXlwh8Oosr6+SH5IZmsjS9xwGrSEmp0d55KEZTrz7Rb57/lXKJZOvf+kS8/tGMI0ai4ubFMcKvPi1LXr1NqniOHv3zHH2jWvMHipQW28TawO+9fY3eP7Ct3jn+jfZ3N4AX6XbUriycpluO0EyJeJYo1nvsL1dxSp0aDbi2x0/QRAEQRAEQRAEQRBugigNhVuu3nJYWewRBwFmVmVkYohySePQgUMoioJtFjh86G7GplSGRizyIzHTE3tRNJmH7r2DhWMml1ef48r5LVwnQg0NFFlj6dpVDu6d4St//gKf/42/Zm15B12zkRSF3Z0uumYxPDRKEquMjpX5kR99ikceepK245IrjlAo5YjjEM0wCPBJZInsUJnZQ3vIKhYXLl7DDSPePr9OENokmJhpEy/0WN9o4vgOURQRhj69Xpv1tR1s28K2bXRDxrQUfN9F02WSJEFVQZYloigmDCT8YECnnmJifApJimi1d8hkMuSLJr4Xsf/gKLp2Y79hv+8QRxKZdI44qWBbs1Rrr7N2LuHSxfNUt9t4LfjmX3yVrbVt5mam+fhHH8GJ14nCmDe+t81k8DSan6bZWuGnPnsPBBN87MOfYe/8YT
```
Comment (mvsusp) [inline sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb]: Please add code to delete the endpoint.

Comment (la-cruche) [reply sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb]: Thanks for the review! Find here new commit taking into account modifications https://github.com/awslabs/amazon-sagemaker-examples/pull/882/commits/b54fd5cc668220a6194a00f864733fb67b706657. Edits are the following:

1. Added code to delete the endpoint
2. Moved all the python `import` statement to the top of notebook
3. Simplified the scatter-plot code to contain only needed functionality
4. Removed the local test of the script mode file, which didn't add value
5. Changed deployment instance from `ml.c5.xlarge` to `ml.t2.medium` to minimize charges and make sure that limit-related issues are unlikely to be met by users.

I ran successfully end-to-end in 3 regions (`us-east-1`, `eu-west-1`, `ap-southeast-2`)

File: sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+    "from sagemaker import get_execution_role\n",
+    "from sagemaker.mxnet.model import MXNetModel\n",
+    "from sagemaker.mxnet.estimator import MXNet\n",
+    "\n",
+    "sess = sagemaker.Session()\n",
+    "\n",
+    "embedding_model = MXNet(\n",
+    "    entry_point='embedding_server.py',\n",
+    "    py_version='py3',\n",
+    "    framework_version='1.3',\n",
```
Comment (yuanzhua) [inline sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb]: can we use 1.4 which is the latest version?

Comment (la-cruche) [reply sagemaker-python-sdk/mxnet_gluon_embedding_server/mxnet_embedding_server.ipynb]: Hi @yuanzhua , thanks for following-up! I actually did a bit more than updating version:

1. I replaced the training API copy part by a tar compression and copy to S3. Faster and simpler.
2. I enriched the documentation
3. I switched to MXNet 1.4

Ran successfully end-to-end in eu-west-1, us-east-1, ap-southeast-2
let me know next steps
cheers

**Pull request merged**